### PR TITLE
Added a "please sign out" page; part of #2049, fixes #2053

### LINKED ISSUE #2053: Error Template doesn't render
While debugging #2049, and bringing down the id and login servers, that Thimble's error templates don't seem to always render.  See also #1797.

> Template render error: (/vagrant/views/error.html)
  Error: template not found: remix-projects.html
    at Object.exports.prettifyError (/vagrant/node_modules/nunjucks/src/lib.js:34:15)
    at /vagrant/node_modules/nunjucks/src/environment.js:486:31
    at eval (eval at <anonymous> (/vagrant/node_modules/nunjucks/src/environment.js:565:24), <anonymous>:52:11)
    at createTemplate (/vagrant/node_modules/nunjucks/src/environment.js:212:25)
    at next (/vagrant/node_modules/nunjucks/src/lib.js:210:13)
    at handle (/vagrant/node_modules/nunjucks/src/environment.js:252:25)
    at /vagrant/node_modules/nunjucks/src/environment.js:263:21
    at next (/vagrant/node_modules/nunjucks/src/lib.js:207:13)
    at Object.exports.asyncIter (/vagrant/node_modules/nunjucks/src/lib.js:214:5)
    at Obj.extend.getTemplate (/vagrant/node_modules/nunjucks/src/environment.js:242:17)

It's weird that this fails to find the `remix-projects.html` template [here](https://github.com/mozilla/thimble.mozilla.org/blob/fa3dde088dc016a562520e62a9df1f5558ba8f70/views/error.html#L63) since we do have it [here](https://github.com/mozilla/thimble.mozilla.org/blob/fa3dde088dc016a562520e62a9df1f5558ba8f70/views/remix-projects.html).  I wonder if it's missing data it needs in one of the places we [render it](https://github.com/mozilla/thimble.mozilla.org/search?utf8=%E2%9C%93&q=render%28+error.html&type=)

### PR DESCRIPTION
**Pull request by flukeout**
If all goes well, should be available at ``/sign-out``.

image

Also fixes the ``error.html`` page, which was missing a template and thus not loading properly.

cc @gideonthomas

### CODE REVIEW

File: views/error.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
           <img class="spilled-thimble" src="/img/spilled-thimble.png" />
           <h1>{{ gettext("errorSomethingWentWrong") }}</h1>
           <h2>
-            {{ gettext("errorSomethingWentWrongDetails") }}
+            {{ gettext("errorSomethingWentWrongDetails") }}"
```
Comment (humphd) [inline views/error.html]: Stray `"` at the end of the line here.

File: views/error.html [hunk 2]
```diff
@@ -60,7 +60,7 @@
       </div>
     </section>
 
-    {% include 'remix-projects.html' %}
+
```
Comment (humphd) [inline views/error.html]: This should fix #2053, I'll update the title of the PR so it auto-closes.

**Comment by humphd** [comment]
I love the look of what you made, nice job.  This is *way* better than before.

**Comment by flukeout** [comment]
@humphd Fixed the stray ``"``

**Comment by flukeout** [comment]
Wait let me push one more string change.

**Comment by gideonthomas** [comment]
https://github.com/flukeout/thimble.webmaker.org/pull/2

**Comment by flukeout** [comment]
@gideonthomas travis if failing on some of the changes you made.

**Comment by gideonthomas** [comment]
sigh, k, can I add a comment and you fix in this PR?

File: server/routes/index.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 var config = require("./config");
+var Middleware = require("../lib/middleware");
```
Comment (gideonthomas) [inline server/routes/index.js]: change to `var middleware = require("../lib/middleware")();`

File: server/routes/index.js [hunk 4]
```diff
@@ -15,7 +16,9 @@ function getPageData(req) {
 
 module.exports = function() {
   return {
-    init: function(app, middleware) {
+    init: function(app) {
+      var middleware = Middleware(config);
```
Comment (gideonthomas) [inline server/routes/index.js]: remove this.

**Comment by flukeout** [comment]
@gideonthomas - made the changes you requested for the ``middleware`` travis problem. R?

**Pull request merged**